docs/en/stack/security/get-started-security.asciidoc

```diff
@@ -24,11 +24,16 @@ running.
 . Launch the {kib} web interface by pointing your browser to port 5601. For 
 example, http://127.0.0.1:5601[http://127.0.0.1:5601].
 
-[role="xpack"]
-[[get-started-license]]
-=== Install a trial license
-
-include::{docdir}/get-started-trial.asciidoc[]
+. Verify that you are using a license that includes the role-based access
```

Comment:
There are now no license types that don't allow security configuration, so I think we can drop this paragraph? Alternatively, we could use this to ensure folks have the default distribution.